template.yaml

```diff
@@ -834,6 +884,391 @@ Resources:
         }
       RoleArn: !GetAtt StateMachineIAMRole.Arn
 
+  GcloudOnBoardingStateMachine:
```

Comment:
Change the name to UKGCloudOnBoardingStateMachine.

Reply:
Have we modified any state of this state machine or is it replica of commercial on-boarding state machine?

Reply:
We have modified only the new state condition for datadog to make it optional rest all is replica of commercial one.